### PR TITLE
Eventing Module build job

### PR DESCRIPTION
**Pull request by muralov**
**Description**
Add Prow jobs to build Eventing module

**Related issue(s)**
[#81](https://github.com/kyma-project/eventing-manager/issues/81)

### CODE REVIEW

File: templates/data/eventing-manager-data.yaml [hunk 1]
```diff
@@ -2,20 +2,117 @@ templates:
...
+                    description: Job to build eventing module for a release.
+                  always_run: true
+                  labels:
+                    preset-sa-kyma-push-images: "true"
+                    preset-bot-github-token: "true"
+                  env:
+                    KUSTOMIZE_VERSION: "v4.5.6"
+                    MODULE_REGISTRY: "europe-docker.pkg.dev/kyma-project/prod/unsigned"
+                    IMG: "europe-docker.pkg.dev/kyma-project/prod/eventing-manager:${PULL_BASE_SHA}"
+                  command: "./scripts/release.sh"
```
Comment (marcobebway) [inline templates/data/eventing-manager-data.yaml:92]: I didn't find this script file [here](https://github.com/kyma-project/eventing-manager).

Comment (muralov) [reply templates/data/eventing-manager-data.yaml:92]: it is [here](https://github.com/muralov/eventing-manager/tree/module-build-job). There is another PR in review containing this script. I added on-hold  to this PR until another one is merged.

File: templates/data/eventing-manager-data.yaml [hunk 2]
```diff
@@ -2,20 +2,117 @@ templates:
   - from: generic.tmpl
     render:
       - to: ../../prow/jobs/eventing-manager/eventing-manager-generic.yaml
+        localSets:
+          job_build:
+            skip_report: "false"
+            max_concurrency: "10"
+            decorate: "true"
+            branches:
+              - "^master$"
```
[the remark this answers is not in the document]
Comment (muralov) [reply templates/data/eventing-manager-data.yaml:11]: I've copy-pasted from [nats prow job templates ](https://github.com/kyma-project/test-infra/blob/f446b51cae022c0215c1d87b37686a173492ce22/templates/data/nats-manager-data.yaml#L11). Should I delete it? I assume there won't any branch names `master`.

[the remark this answers is not in the document]
Comment (marcobebway) [reply templates/data/eventing-manager-data.yaml:11]: Let's keep it for now, and later we can remove it from both nats-manager and eventing-manager.

**Review by marcobebway — APPROVED** [review]

**Pull request merged**